### ISSUE TITLE
Migrate this to Community Plugins?

### ISSUE THREAD

**Issue by jmezach** [TRIMM/plugin-dependencytrack#109]
I was wondering if it would perhaps be beneficial to migrate this plugin to the [Backstage Community Plugins](https://github.com/backstage/community-plugins) repository. That way it would be kept up-to-date a bit better and the community could perhaps add new features.

**Comment by harmenbrinkmanTRIMM** [comment]
Hey @jmezach, we will be migrating this plugin to a Community Plugin. Will let you know once this has been done :)

**Comment by cal5barton** [comment]
@harmenbrinkmanTRIMM any progress made on moving that to community plugins?

**Comment by harmenbrinkmanTRIMM** [comment]
@jmezach @cal5barton Yes! PR can be found here: https://github.com/backstage/community-plugins/pull/2534

**Comment by jmezach** [comment]
@ronaldtb @harmenbrinkmanTRIMM This was closed but the plugin never got merged into the community-plugins repo. This plugin hasn't been updated for a long time now. Still valuable imho.
